app/facades/quotable_facade.rb

```diff
@@ -0,0 +1,15 @@
+class QuotableFacade
+
+  def self.search_for_quotes(query)
+    # require "pry"; binding.pry
+    json = QuotableService.search_quotes(query)
+    if json[:results].count > 0
+      json[:results][0..9].map {|quote| Quote.new(quote, json[:count])}
```

Comment:
Nice solution to include the count in your poro. It's much more efficient to collect all the data you need at one time rather than making multiple API calls. Good work!